lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -84,6 +84,29 @@ class CategoryRenderer {
     this.dom.find('.lh-audit__description', auditEl)
         .appendChild(this.dom.convertMarkdownLinkSnippets(audit.result.description));
 
+    if (audit.result.stackPacks) {
+      Object.entries(audit.result.stackPacks).forEach(([name, pack]) => {
+        const packElm = this.dom.createElement('div');
+        packElm.classList.add('lh-audit__description');
+        packElm.style.display = 'flex';
+        packElm.style.alignItems = 'center';
+
+        const packElmImg = this.dom.createElement('img');
+        packElmImg.src = pack.icon;
+        packElmImg.alt = name;
+        packElmImg.style.maxWidth = '50px';
+        packElmImg.style.marginRight = '15px';
+
+        const packElmSpan = this.dom.createElement('span');
```

Comment:
`convertMarkdownLinkSnippets` does return an `HTMLSpanElement`, does that suffice?